src/components/app/matching/MatchingCockpit.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import { MatchesList } from "./MatchesList";
+import { MatchProps } from "./Match";
+
+interface MatchingCockpitState {
+  matches: Array<MatchProps>
+}
+
+const MATCHES_PLACEHOLDER = {
+  matches: [
+    {
+      matchName: "Tomas",
+      chatStatusIcon: "Chatty",
+      matchIsStarred: true,
+      matchPicture: "awesome"
+    } as MatchProps
+  ]
+} as MatchingCockpitState;
+
+class MatchingCockpit extends React.Component<any, MatchingCockpitState> {
+
+  state = MATCHES_PLACEHOLDER;
+
+  render() {
+    return (
+      <div>
+        <h1>Matches</h1>
+        <MatchesList matches={this.state.matches}/>
+        <h1>Candidates Stack</h1>
+        <p>Candidate Stack placeholder</p>
+      </div>
+    )
+  }
+}
+
+export default MatchingCockpit;
```

Comment:
see above regarding the default export

also, when exporting anything, please put the `export` statement next to the exported element, i.e. `export class MatchingCockpit`

Reply:
ok